src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -207,7 +199,7 @@ export class RetireJsEngine extends AbstractRuleEngine {
 
 	private async executeRetireJs(invocation: RetireJsInvocation, verboseViolations: boolean): Promise<RuleResult[]> {
 		return new Promise<RuleResult[]>((res, rej) => {
-			const cp = cspawn(RetireJsEngine.NODE_EXEC_PATH, invocation.args);
+			const cp = cspawn('npx', ['retire'].concat(invocation.args));
```

Comment:
Thoroughly tested this. Since we have made retire a dependency, npx knows to look at the local node_modules folder first. Also note that the exported functions from retire do not seem be documented (nor obvious how to use) so we are best using the retire cli still.